### PR TITLE
react script fix

### PR DESCRIPTION
**Pull request by JohannesDoberer**

### CODE REVIEW

File: scripts/setup/react.sh [hunk 1]
```diff
@@ -16,8 +16,8 @@ npx create-react-app $folder && cd $folder
 echo yes | npm run eject
 
 # install dependencies
-npm i -P @luigi-project/core @luigi-project/client fundamental-styles @sap-theming/theming-base-content react-router-dom
-npm i copy-webpack-plugin@5 webpack@4.43.0 webpack-cli@3.3.12 @babel/core @babel/preset-env babel-loader --save-dev
+npm i -P @luigi-project/core @luigi-project/client fundamental-styles@0.11.0 @sap-theming/theming-base-content react-router-dom
+npm i copy-webpack-plugin@5 webpack webpack-cli @babel/core @babel/preset-env babel-loader --save-dev
```
Comment (stanleychh) [inline scripts/setup/react.sh:20]: We need a specific version of webpack, webpack-cli at angluar.sh and vue.sh. But not here, why?

Comment (JohannesDoberer) [reply scripts/setup/react.sh:20]: https://github.com/SAP/luigi/issues/1682

**Review by stanleychh — APPROVED** [review]
Tested and worked for me.

**Review by UlianaMunich — APPROVED** [review]
LGTM!

**Pull request merged**